server/layout.go

```diff
@@ -23,6 +23,23 @@ func newLayoutResponse(layout chronograf.Layout) layoutResponse {
 	httpAPILayouts := "/chronograf/v1/layouts"
 	href := fmt.Sprintf("%s/%s", httpAPILayouts, layout.ID)
 	rel := "self"
+
+	for idx, cell := range layout.Cells {
+		axes := []string{"x", "y", "y2"}
+
+		if cell.Axes == nil {
+			layout.Cells[idx].Axes = make(map[string]chronograf.Axis, len(axes))
+		}
+
+		for _, axis := range axes {
+			if _, found := cell.Axes[axis]; !found {
+				layout.Cells[idx].Axes[axis] = chronograf.Axis{
+					Bounds: []string{},
```

Comment:
So this has to do with how Go defines [zero values in the spec](https://golang.org/ref/spec#The_zero_value). From that page (emphasis mine):

> Each element of such a variable or value is set to the zero value for its type: false for booleans, 0 for integers, 0.0 for floats, "" for strings, and **nil for pointers, functions, interfaces, slices, channels, and maps**.

So if we want the response to have `bounds: []`, we need to explicitly allocate a slice here.